source/css/sass/mixins/_typography.scss

```diff
@@ -109,3 +109,9 @@
 @mixin label-tag-typography() {
   @include _label-typography($color-primary-normal);
 }
+
+@mixin remove-bullets-unordered-list() {
+  padding-inline-start: 0;
```

Comment:
Need to use `@include padding(0, "inline-start");` to make sure fallbacks are added.